renderdoc/driver/vulkan/vk_common.h

```diff
@@ -936,6 +943,7 @@ DECLARE_DESERIALISE_TYPE(VkTextureLODGatherFormatPropertiesAMD);
 DECLARE_DESERIALISE_TYPE(VkValidationCacheCreateInfoEXT);
 DECLARE_DESERIALISE_TYPE(VkValidationFlagsEXT);
 DECLARE_DESERIALISE_TYPE(VkWriteDescriptorSet);
+DECLARE_DESERIALISE_TYPE(VkConditionalRenderingBeginInfoEXT);
```

Comment:
Understood. There is nothing to change here, right?

Reply:
Ah yes, sorry you have all the `DECLARE_DESERIALISE_TYPE` needed. I misread when going through the code because this one is out of order - I've kept the other structs in alphabetical order to keep things easier to find, I thought this was with the second smaller list of structs that don't have a `pNext` chain but _do_ have some array or dynamic data which needs to be deallocated.

So I think just moving the added entries into alphabetical order in both lists (`DECLARE_REFLECTION_STRUCT` and `DECLARE_DESERIALISE_TYPE`) would do the trick. Note the other two are closer, but not quite alphabetical sorted.